gooddata-sdk/gooddata_sdk/catalog/data_source/declarative_model/data_source.py

```diff
@@ -49,6 +54,25 @@ def from_dict(cls, data: dict[str, Any], camel_case: bool = True) -> CatalogDecl
         declarative_data_sources = DeclarativeDataSources.from_dict(data, camel_case)
         return cls.from_api(declarative_data_sources)
 
+    @classmethod
```

Comment:
There should not be classmethod. Move it out of class or make it staticmethod. In case you choose to use staticmethod, please, make also get_pdm_folder in  pdm.py staticmethod.

Reply:
classmethod converted into static method.
get_pdm_folder can't be static method, because relying method load_from_disk must stay static, so we can call it from the outside without instantiating it.